lib/segment/Cargo.toml

```diff
@@ -39,6 +39,7 @@ num-derive = "0.3.3"
 rand = "0.8"
 bit-vec = "0.6"
 seahash = "4.1.0"
+json-patch = "0.2.6"
```

Comment:
Is this used somewhere?

Reply:
here https://github.com/qdrant/qdrant/blob/d564468ff7a3bf53bbf7186288f06ab5a8f4dfe0/lib/segment/src/types.rs#L330